### PR TITLE
Fix types

### PR DESCRIPTION
**Pull request by guillemcordoba**

### CODE REVIEW

File: src/api/admin.ts [hunk 1]
```diff
@@ -214,8 +220,8 @@ type DnaManifest = {
 }
 
 export type DnaBundle = {
-    manifest: DnaManifest,
-    resources: ResourceMap,
+    manifest: DnaManifest;
```
Comment (zippy) [inline src/api/admin.ts:223]: Is this really supposed to be a ;?

Comment (guillemcordoba) [reply src/api/admin.ts:223]: Yeap! Actually autoformatting with prettier fixes a lot of format issues that exist in this file at the moment. I could commit those if you wanted.

Comment (zippy) [reply src/api/admin.ts:223]: That would be great because then it's standardized.  Thanks!

**Review by zippy — APPROVED** [review]
Thanks for catching all these mistakes!

**Pull request merged**